### ISSUE TITLE
Permission problems.

### ISSUE THREAD

**Issue by CuteVampire** [msummers123/Movecraft-3#275]
I am using Minecraft 1.10.2 and Movecraft 6.0. Every time I right click the sign for piloting a ship I have no issues, but when I use a stick to get the ship remove it comes up with "Insufficient Permissions". I have use 2 separate permission plugins and have yet to see that issue go away. Is there a trick to this?

**Comment by mwkaicz** [comment]
No, that trick is called "set permissions correctly" ;) two perms plugins is way to the hell

**Comment by CuteVampire** [comment]
Well I never used two at a time. I've looked at multiple guides on how to
set them up with no luck. Not even oping myself works
